### DISCUSSION TITLE
Battle Skill Activation Order

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#187] [Q&A]
Is it possible to make the little popup for a skill activation display on the opposite side of the round it triggers on?

Use case: I'm making counter (a skill that reflects physical damage) by piggybacking off of the devil axe functionality in `BattleHit.c`. However, this will obviously trigger on the skill holder's turn which causes the activation animation and popup to display on that side too, instead of the skill holder's side. I can see this being confusing for users. Is there a way around this, or am I better off ust disabling the animation and popup?

![image](https://github.com/MokhaLeee/fe8u-cskillsys-kernel/assets/164426396/de705ff4-5569-46b4-8e33-8db2bf526097)

**Comment by MokhaLeee** [comment]
Although it can be modified, I think it is obvious that the scene you mentioned should be displayed by the target side of the skill animation. Am I wrong in understanding?

**Reply by JesterWizard** [reply]
I'm not sure it is obvious. Is there a single other case of an activation rate skill where the effect applies to the target on their turn? That would be helpful as reference.

What would be obvious to me is to have the skill animation and popup appear on the right (where the actor is) before the effect is then applied on the target's turn. 

I'm not too bothered about it though, as worst case scenario, I can just hide the skill activation animation/popup as it's a 100% activation rate anyway.

**Comment by MokhaLeee** [comment]
EfxSkill is not designed to support such features yet

**Reply by JesterWizard** [reply]
Yeah, I had a feeling. 

In which case I'll hide the animation for now. Thanks for letting me know 👍

**Reply by JesterWizard** [reply]
Coming back to this, it very much is possible to do this, I just needed to use `RegisterTargetEfxSkill` instead of `RegisterActorEfxSkill`

**Reply by MokhaLeee** [reply]
why
